kubernetes/cmsweb/scripts/test_k8s.sh

```diff
@@ -18,7 +18,7 @@ fi
 # scram architecture
 arch=${SCRAM_ARCH:-slc7_amd64_gcc700}
 
-urls="$url/couchdb $url/acdcserver/_all_docs $url/reqmgr2/data/info $url/dbs/$inst/global/DBSReader/datasets?dataset=/ZMM*/*/* $url/das $url/workqueue/index.html $url/phedex $url/phedex/datasvc/doc $url/t0_reqmon/data/info $url/crabserver/preprod/info $url/crabcache/info $url/wmstatsserver/data/info $url/wmstats/index.html $url/t0wmadatasvc/replayone/hello $url/exitcodes $url/wmarchive/data $url/ms-transferor/data/status $url/ms-monitor/data/status $url/ms-output/data/status $url/ms-rulecleaner/data/status"
+urls="$url/couchdb $url/acdcserver/_all_docs $url/reqmgr2/data/info $url/dbs/$inst/global/DBSReader/datasets?dataset=/ZMM*/*/* $url/das $url/workqueue/index.html $url/phedex $url/phedex/datasvc/doc $url/t0_reqmon/data/info $url/crabserver/preprod/info $url/crabcache/info $url/wmstatsserver/data/info $url/wmstats/index.html $url/t0wmadatasvc/replayone/hello $url/exitcodes $url/wmarchive/data $url/ms-transferor/data/status $url/ms-monitor/data/status $url/ms-output/data/status $url/ms-rulecleaner/data/status $url/ms-unmerged/data/status?rses=t1 $url/ms-unmerged/data/status?rses=t2-t3 $url/ms-unmerged/data/status?rses=t2-t3-us"
```

Comment:
Please update it according to the frontend rules:
https://github.com/dmwm/deployment/blob/master/frontend/backends-k8s-preprod.txt#L41-L43

thus, instead of `rses=t2-t3`, it should be `rse_type=t2t3` and so on.

Reply:
adjusted.